content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="http://dtdg.co/code-quality" header="false" >}}
+Quality Gates are in private beta. To request access, complete the form.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog. You can create two types of rules: Static Analysis and Tests.
+For example, you can block code from being merged if it introduces new [flaky tests][8] or code style violations; issues that wouldn't normally fail your CI/CD pipelines and end up deployed to production.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed.
+This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents and unwanted behaviors.
```

Comment:
I think insert comma after "reducing incidents"

Reply:
Mmm some sort of Oxford comma? I don't know, it sounds better without the comma to me as both things start from "reducing"

Reply:
```suggestion
This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents, and minimizing unwanted behaviors.
```